src/api/cli/cli.go

```diff
@@ -28,14 +28,16 @@ const (
 	defaultWalletName = "$COIN_cli" + walletExt
 	defaultWalletDir  = "$HOME/.$COIN/wallets"
```

Comment:
the DATA_DIR will affect the defaultWalletDir, it will be `$DATA_DIR/wallets` instead